voluncheer/profiles/forms/gallery.py

```diff
@@ -0,0 +1,30 @@
+from django import forms
+from django.db import transaction
+
+from profiles.models import GallaryPost
```

Comment:
Gallary should be spelled "Gallery," here and in the rest of the file.

Reply:
Fixed